packages/typespec-test/test/eventgrid_modular/generated/typespec-ts/src/index.ts

```diff
@@ -10,14 +14,17 @@ export {
   FailedLockToken,
   ReleaseResult,
   RejectResult,
-} from "./api/models.js";
-export {
+  publishCloudEvent,
+  publishCloudEvents,
+  receiveCloudEvents,
+  acknowledgeCloudEvents,
+  releaseCloudEvents,
+  rejectCloudEvents,
```

Comment:
I don't think we should export the operations in the api layer to the top classical client layer, right? @bterlson  @mpodwysocki